style.css

```diff
@@ -623,3 +622,150 @@ header {
   margin-top: -25px;
   background: #dfe1e6;
 }
+
+/* Contact Section */
+.contact_container {
+  position: absolute;
+  width: 375px;
+  height: 784px;
+  left: 0;
+  top: 4470px;
+  background: #6070ff;
+  background-image: url(images/bg/contact\ form\ shapes-mobile@2x.png);
+  transform: matrix(1, 0, 0, 1, 0, 0);
+  border-top-left-radius: 60px;
+  background-repeat: no-repeat;
+  background-size: 216px;
+  background-position: right;
+}
```

Comment:
- [ ] Ouch! this is so close but as specified in the project requirements please make sure to use `flexbox` to lay out the elements this will give you better options in responsiveness. ✔
- Here's the project requirement as a reminder:
  ![image](https://user-images.githubusercontent.com/104968236/235421571-31f08542-aa81-4966-bd9b-c78ccb766aac.png)

- [ ] [OPTONAL] As I've advised you before please restrain from using absolute positioning and fixed width & height as this will make it very difficult to layout the elements when you scale up to desktop.